### PR TITLE
feat: add navbar component + routing

### PR DESCRIPTION
**Pull request by pankaj-prog**
- added navbar component 
skill up- navbar and routing

- added routing functionality to the navbar using react-router

### CODE REVIEW

File: src/components/Navbar/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <div id="header-ctas" className="header-ctas">
+          <NavLink to="/wishlist" className="btn icon-btn link">
+            <i className="far fa-heart"></i>
+          </NavLink>
+          <NavLink to="/cart" className="btn icon-btn link">
+            <i className="fas fa-shopping-cart"></i>
+          </NavLink>
+          <NavLink to="/signin" className="link">
+            <button className="btn btn-solid-primary btn-rc">Sign-in</button>
+          </NavLink>
```
Comment (aiman-ansari) [inline src/components/Navbar/Navbar.jsx:43]: Good that instead of Link you used here NavLink for nav links.

**Review by aiman-ansari — APPROVED** [review]
- Good folder structure
- Neat and clean code
- Good to merge:)

File: src/pages/Wishlist/Wishlist.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const Wishlist = () => {
+  return <h1>This is Wishlist page</h1>;
+};
```
Comment (gauravsinhaweb) [inline src/pages/Wishlist/Wishlist.jsx:5]: - nice use of es6

File: src/pages/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export { default as Cart } from "./Cart/Cart";
+export { default as Home } from "./Home/Home";
+export { default as Wishlist } from "./Wishlist/Wishlist";
+export { default as Courses } from "./Courses/Courses";
+export { default as SignIn } from "./auth/SignIn/SignIn";
```
Comment (gauravsinhaweb) [inline src/pages/index.js:5]: - good use of exporting components!

**Review by gauravsinhaweb — APPROVED** [review]

**Review by m26unkwn — APPROVED** [review]
Things which are good.
1. Using NAVLINK instead of LINK😎
2. Naming of Components ✏
4. Folder structure 📚

File: src/pages/index.js (hunk 3 above)
Comment (pankaj-prog) [reply src/pages/index.js:5]: Thankyou

**Pull request merged**